src/odemis/util/dataio.py

```diff
@@ -62,15 +62,19 @@ def data_to_static_streams(data):
         ci = dims.find("C")  # -1 if not found
         if (((model.MD_WL_LIST in d.metadata or
               model.MD_WL_POLYNOMIAL in d.metadata) and
-             (ci >= 0 and d.shape[ci] > 1)
-             ) or
-            (ci >= 0 and d.shape[ci] >= 5)
-           ):
-            # Spectrum: either it's obvious according to metadata, or no metadata
-            # but lots of wavelengths, so no other way to display
-            name = d.metadata.get(model.MD_DESCRIPTION, "Spectrum")
-            klass = stream.StaticSpectrumStream
-        elif model.MD_PIXEL_DUR in d.metadata and ti >= 0 and d.shape[ti] > 1:
+             (ci >= 0 and d.shape[ci] > 1)) or
+             (ci >= 0 and d.shape[ci] >= 5)):  # TODO if MD_WL_LIST but streak image in "Focus" mode
```

Comment:
I think you can drop this TODO. As, the one just bellow somehow implies that if there is not MD_TIME_LIST, it's the StaticStream's problem. And so it should work fine the streak ccd acquired data while not streaking took place.